src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -119,6 +119,7 @@ class LicenceService(
     return createLicenceResponse
   }
 
+  @Transactional
```

Comment:
this is required as there needs to be a session to lazily load the related entities when building up the detailed DTO response

Previously it was relying on retrieving everything through eager loading